.github/workflows/release.yml

```
@@ -131,3 +132,62 @@
+        with:
+          push: ${{ secrets.DOCKERHUB_USERNAME != '' }}
+          tags: jackalcooper/beaver-livebook-${{ matrix.image.suffix }}:latest
+          cache-from: type=registry,ref=jackalcooper/beaver-livebook-${{ matrix.image.suffix }}:latest
+          cache-to: type=inline
+          platforms: |
+            ${{ matrix.image.platform }}
+          file: docker/livebook.dockerfile
+          target: base
+          build-args: |
+            ZIG_URL=${{ matrix.image.zig }}
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)